Resources/Prototypes/Catalog/Fills/Crates/salvage.yml

```diff
@@ -20,6 +20,7 @@
       - id: Wirecutter
       - id: ClothingBeltUtility
       - id: OreBag
+      - id: ClothingNeckCloakSalvager
```

Comment:
make this a prob spawn in salv lockers rather than adding it to a crate

Reply:
Changed spawn.